activerecord/lib/active_record/query_logs.rb

```diff
@@ -26,6 +26,7 @@ module ActiveRecord
   # * +socket+
   # * +db_host+
   # * +database+
+  # * +line+
```

Comment:
Also I wonder if `line` really makes sense as a name, as it's not just the line, but the whole location. Maybe `location` or `source_location`?

Reply:
I named it after the marginalia's `:line`, but yeah, changed it to `:source_location`.
CI is still red, but looks like is not related to this PR.